sys/kern/lockdep.c

```diff
@@ -250,6 +250,7 @@ void lockdep_acquire(lock_class_mapping_t *lock) {
   lockdep_unlock();
 }
 
+/* This function is called with the `lock` held. */
```

Comment:
If this comment is supposed to address my remarks about interrupts, this is not what I meant -- I wanted the comment to say that acquiring the lockdep lock here is unnecessary, but only because interrupt handlers can only use spinlocks, and spinlocks are not tracked by lockdep. If spinlocks were tracked by lockdep, we could get an interrupt in the middle of `lockdep_release()` and the interrupt handler could see an inconsistent state.

Reply:
OK, I get it now. Spinlocks will be added in the near future, not in this PR, so I will just put the locks back.